bitnami/mariadb/templates/update-password/job.yaml

```diff
@@ -0,0 +1,241 @@
+      {{- end }}
+      automountServiceAccountToken: {{ .Values.passwordUpdateJob.automountServiceAccountToken }}
+      {{- if .Values.passwordUpdateJob.hostAliases }}
+      hostAliases: {{- include "common.tplvalues.render" (dict "value" .Values.passwordUpdateJob.hostAliases "context" $) | nindent 8 }}
+      {{- end }}
+      initContainers:
+        {{- if .Values.passwordUpdateJob.initContainers }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.passwordUpdateJob.initContainers "context" $) | nindent 8 }}
+        {{- end }}
+      containers:
+        - name: update-credentials
+          image: {{ template "mariadb.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          {{- if .Values.passwordUpdateJob.command }}
+          command: {{- include "common.tplvalues.render" (dict "value" .Values.passwordUpdateJob.command "context" $) | nindent 12 }}
+          {{- else }}
+          command:
+            - /bin/bash
+            - -ec
+          {{- end }}
+          {{- if .Values.passwordUpdateJob.args }}
+          args: {{- include "common.tplvalues.render" (dict "value" .Values.passwordUpdateJob.args "context" $) | nindent 12 }}
+          {{- else }}
+          args:
+            - |
+              {{- if .Values.usePasswordFiles }}
+              # We need to load all the secret env vars to the system
+              for file in $(find /bitnami/mariadb/secrets -type f); do
+                  env_var_name="$(basename $file)"
+                  echo "Exporting $env_var_name"
+                  export $env_var_name="$(< $file)"
+              done
+              {{- end }}
+              primary_host={{ include "mariadb.primary.fullname" . }}-0.{{  printf "%s-headless" (include "common.names.fullname" .) | trunc 63 | trimSuffix "-" }}
+              echo "Starting password update job"
+              if [[ -f /job-status/root-password-changed ]]; then
+                echo "Root password already updated. Skipping"
+              else
+                echo "Updating root password"
+                mysql -uroot -p$MARIADB_PREVIOUS_ROOT_PASSWORD -h$primary_host --port={{ .Values.primary.containerPorts.mysql }} -e "SET PASSWORD for 'root'@'%' = PASSWORD('$MARIADB_NEW_ROOT_PASSWORD');"
+                touch /job-status/root-password-changed
+                echo "Root password successfully updated"
+              fi
+              {{- if not (empty .Values.auth.username) }}
+              if [[ -f /job-status/password-changed ]]; then
+                echo "User password already updated. Skipping"
+              else
+                echo "Updating user password"
+                mysql -u$MARIADB_USER -p$MARIADB_PREVIOUS_PASSWORD -h$primary_host --port={{ .Values.primary.containerPorts.mysql }} -e "SET PASSWORD for '$MARIADB_USER'@'%' = PASSWORD('$MARIADB_NEW_PASSWORD');"
```

Comment:
You can potentially load MariaDB libraries and use wrappers such as `mysql_remote_execute`:
- https://github.com/bitnami/containers/blob/main/bitnami/mariadb/11.5/debian-12/rootfs/opt/bitnami/scripts/libmariadb.sh#L697
And replace `echo` references with `info/debug` ones

Reply:
Done